praw/models/reddit/subreddit.py

```diff
@@ -2030,6 +2030,23 @@ def __iter__(
         for template in self.subreddit._reddit.get(url):
             yield template
 
+    def get(
```

Comment:
The method name of `get` doesn't seem right here. Maybe `user_selectable` or something more descriptive.